### DISCUSSION TITLE
Who pays commission for default ETH pool settings?

### DISCUSSION THREAD

**Discussion by kr1z1s** [coinfoundry/miningcore#1214] [Q&A] [answered]
Hi!

I have a working ETH pool.

The payout settings are taken from standard sample configuration file for ETH.

Minimum amount for payment was initially 0.05. I lowered it to 0.01 and restarted pool. 

Payments have started. But... pool balance was not enough for the last payment. 
So the question is. 
Does... miner or pool pay commission?

**Comment by oliverw** [comment, marked as answer]
https://github.com/oliverw/miningcore/blob/master/src/Miningcore/Blockchain/Ethereum/Configuration/EthereumPoolPaymentProcessingConfigExtra.cs
